android/build.gradle

```diff
@@ -109,6 +109,13 @@ static def getClientId(rootProject) {
     return rootProject.clientId
 }
 
+static def getHyperAssetVersion(rootProject) {
+    if(!rootProject.hasProperty('assetVersion')) {
```

Comment:
call the ext property as hyperAssetVersion

Reply:
done